R/render-plot.R

```diff
@@ -610,6 +610,10 @@ find_panel_info_api <- function(b) {
   coord  <- ggplot2::summarise_coord(b)
   layers <- ggplot2::summarise_layers(b)
 
+  `%NA_OR%` <- function(x, y) {
+    if (is.na(x)) y else x
```

Comment:
?rlang::is_na

> ‘is_na()’ is a scalar predicate and always returns a scalar
     boolean, ‘TRUE’ or ‘FALSE’. If its input is not scalar, it returns
     ‘FALSE’.

```suggestion
    if (is_na(x)) y else x
```

Reply:
FWIW, in practice, the `x` that gets passed to this function will only ever be a single-element vector. (Note the scoping for this function.) If it's not a single-element vector, it's probably good if we find out about it by having a warning occur.

Reply:
Although... I suppose that given the name of the function, it does make sense to look specifically for a single NA using `is_na()`.